tests/reducers/test.app-reducer.js

```diff
@@ -10,6 +10,7 @@ describe('App Reducer', function() {
     return {
       error: {
         debugMessage: 'some informative message',
+        userMessage: undefined,
```

Comment:
I was scratching my head on this but `assert.deepEqual` was failing without it. Maybe it looks at object keys first but it's curious how the key is available when its value is undefined.